### ISSUE TITLE
EnergyPlus does not allow starting at non-zero simulation time

### ISSUE THREAD

**Issue by mwetter** [lbl-srg/modelica-buildings#1870]
If a simulation in Modelica starts at non-zero time, then EnergyPlus fails because it requests an event time that assumes `t=0` as the start time. The error message for
```
RunScript("/home/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-buildings/Buildings/Resources/Scripts/Dymola/ThermalZones/EnergyPlus/Validation/OneZoneOneOutputVariableSummer.mos");
```
is
```
Error: The following error was detected at time: 18748800
EnergyPlus requested at time = 18748800.000000 a next event time of 1200.000000 for modelicaInstance = OneZoneOneOutputVariableSummer.zon. Zero time steps are not supported. Check with support.
The stack of functions is:
Buildings.ThermalZones.EnergyPlus.BaseClasses.zoneExchange
Buildings.ThermalZones.EnergyPlus.BaseClasses.zoneExchange(
zon.fmuZon.adapter, 
initial(), 
zon.heaPorAir.T, 
zon.ports[1].Xi_outflow[1], 
zon.fmuZon.mInlet_flow, 
zon.fmuZon.TAveInlet, 
zon.fmuZon.QGaiRad_flow, 
zon.fmuZon.AFlo, 
Buildings.ThermalZones.EnergyPlus.BaseClasses.FMUZoneAdapter.round(time, 0.001))

FixInitials:Init: Integrator failed to start model.
```
The example is on branch `issue1870_nonZeroStart_spawn`

**Comment by kbenne** [comment]
@mwetter Fixed in https://github.com/NREL/Spawn/commit/199f7350bb62c32f03c4a16906b23ef827ce0210

**Comment by mwetter** [comment]
Merged to `issue1863_spawn_eplus_9_3` in 09ddeaae9
